motion_dualtask/sub-07/ses-stand/motion/sub-07_ses-stand_task-dualWalking_tracksys-PhaseSpace1_motion.json

```diff
@@ -0,0 +1,23 @@
+{
+	"ACCELChannelCount":0,
```

Comment:
Are fields typically ordered alphabetically or by related topic? Is there any recommendation? I know this is not strictly enforced

Reply:
Correct, they are alphabetically ordered. There was not much intention behind this to be honest, and it was the way it was output by jsonlab application

Reply:
There are no rules about this in BIDS. I'd say an alphabetic ordering is fine (perhaps even more transparent than some grouping that is not officially defined)